src/ucp/rndv/rndv.h

```diff
@@ -12,21 +12,35 @@
 #include <ucs/datastruct/ptr_map.h>
 
 
-enum ucp_rndv_rts_flags {
-    UCP_RNDV_RTS_FLAG_TAG = UCS_BIT(0),
-    UCP_RNDV_RTS_FLAG_AM  = UCS_BIT(1)
-};
+typedef enum {
+    /* RNDV operation statuses, used only by old version for TAG protocol,
+     * kept here for wire compatibility
+     */
+    UCP_RNDV_RTS_FLAG_TAG_COMPAT_VALUE_OK       = UCS_OK,
+    UCP_RNDV_RTS_FLAG_TAG_COMPAT_VALUE_CANCELED = (uint8_t)UCS_ERR_CANCELED,
+    UCP_RNDV_RTS_FLAG_TAG                       = UCS_BIT(0),
+    UCP_RNDV_RTS_FLAG_AM                        = UCS_BIT(1)
```

Comment:
pls add a `UCP_RNDV_RTS_FLAG_MAX = UCS_BIT(4)` and comment about compatibility reserved bits:
```
(gdb) p /t (uint8_t)UCS_ERR_CANCELED
$3 = 11110000
```

Reply:
we reserve `240` value for `UCS_ERR_CANCELED`, not UCS_BIT(4)

Reply:
done